questions/Predict the output

```diff
@@ -0,0 +1,30 @@
+### // interviewer: what will the following code output?
```

Comment:
Wrong formatting, please see other snippets with the structure you had in mind. Remove the `// interviewer` part.